hoverctl/wrapper/config.go

```diff
@@ -177,6 +185,10 @@ func (this Config) BuildFlags() Flags {
 		flags = append(flags, "-tls-verification=false")
 	}
 
+	if this.HoverflyUpstreamProxy != "" {
```

Comment:
Yeah, these are the two common ways that you'd check a string to be empty. Both styles appear throughout the standard Golang packages.

We should probably stick to one of the two styles.